src/main/support.js

```diff
@@ -9,30 +9,26 @@ const arrayify = require("../utils/arrayify");
 const currentVersion = require("../../package.json").version;
 const coreOptions = require("./core-options").options;
 
-function getSupportInfo(version, opts) {
-  opts = {
-    plugins: [],
-    showUnreleased: false,
-    showDeprecated: false,
-    showInternal: false,
-    ...opts
-  };
-
-  if (!version) {
-    // pre-release version is smaller than the normal version in semver,
-    // we need to treat it as the normal one so as to test new features.
-    version = currentVersion.split("-", 1)[0];
+function getSupportInfo(
+  _,
```

Comment:
Can we remove this parameter, and wrap in the export?

Reply:
That would break type inference and overall I think the code is cleaner (less confusing) the way it is now.

Reply:
Nevertheless, I did something with this.